### ISSUE TITLE
Error "The specified prefix was null." When using IBM jvm in Eclipse

### ISSUE THREAD

**Issue by andreabisiach** [Microsoft/team-explorer-everywhere#180]
Hello,

Running Eclipse Neon.2 Release (4.6.2) with 2 Plugin-ins:

**WebSphere® Application Server Developer Tools for Eclipse**
Version: 16.0.0.4
Build id: 20161213_2112
**Team Explorer Everywhere**
Version 14.111.1

When I try to expand "All repostories" inside the perspective "Team Explorer" I receive an error
"Exception creating 'All repositories': The specified prefix was null."

I have tracked down that this problem has started occurring since I changed the jvm used to run eclipse.
My eclipse.ini contains:
`-vm
C:\IBM\WebSphere\AppServer\java\jre\bin\javaw.exe
`
The above is needed to be able to control IBM Websphere Server from inside eclipse. If I comment it out, Team Explorer Everywhere works fine again.

The eclipse log shows this stack

`
!ENTRY com.microsoft.tfs.client.common 2 0 2017-03-09 16:48:50.654
!MESSAGE 
!STACK 0
java.lang.IllegalArgumentException: The specified prefix was null.
	at com.ibm.xml.xlxp.api.stax.msg.StAXMessageProvider.throwIllegalArgumentException(StAXMessageProvider.java:42)
	at com.ibm.xml.xlxp.api.stax.XMLStreamWriterBase.beginStartElement(XMLStreamWriterBase.java:170)
	at com.ibm.xml.xlxp.api.stax.XMLStreamWriterBase.writeStartElement(XMLStreamWriterBase.java:373)
	at com.ibm.xml.xlxp.api.stax.XMLOutputFactoryImpl$XMLStreamWriterProxy.writeStartElement(XMLOutputFactoryImpl.java:123)
	at com.microsoft.tfs.core.ws.runtime.stax.StaxUtils.copyCurrentElement(StaxUtils.java:107)
	at com.microsoft.tfs.core.ws.runtime.types.StaxAnyContentType.readFromElement(StaxAnyContentType.java:215)
	at ms.tfs.workitemtracking.clientservices._03._ClientService5Soap_GetMetadataEx2Response.readFromElement(_ClientService5Soap_GetMetadataEx2Response.java:141)
	at ms.tfs.workitemtracking.clientservices._03._ClientService5Soap12Service$34.readSOAPResponse(_ClientService5Soap12Service.java:874)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequestInternal(SOAPService.java:614)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequest(SOAPService.java:444)
	at ms.tfs.workitemtracking.clientservices._03._ClientService5Soap12Service.getMetadataEx2(_ClientService5Soap12Service.java:864)
	at com.microsoft.tfs.core.clients.workitem.internal.metadata.Metadata.update(Metadata.java:479)
	at com.microsoft.tfs.core.clients.workitem.internal.metadata.Metadata.update(Metadata.java:458)
	at com.microsoft.tfs.core.clients.workitem.internal.WITContext.<init>(WITContext.java:101)
	at com.microsoft.tfs.core.clients.workitem.WorkItemClient.getContext(WorkItemClient.java:394)
	at com.microsoft.tfs.core.clients.workitem.WorkItemClient.getProjects(WorkItemClient.java:337)
	at com.microsoft.tfs.client.common.ui.teamexplorer.TeamExplorerContext.getCurrentProject(TeamExplorerContext.java:103)
	at com.microsoft.tfs.client.eclipse.ui.egit.teamexplorer.TeamExplorerGitRepositoriesSection.initialize(TeamExplorerGitRepositoriesSection.java:95)
	at com.microsoft.tfs.client.eclipse.ui.egit.teamexplorer.TeamExplorerGitRepositoriesSection.initialize(TeamExplorerGitRepositoriesSection.java:105)
	at com.microsoft.tfs.client.common.ui.controls.teamexplorer.TeamExplorerBaseControl$SectionLoadJob.run(TeamExplorerBaseControl.java:481)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
`

java -version shows the following:

`C:\IBM\WebSphere\AppServer\java\jre\bin>java.exe -version
java version "1.8.0"
Java(TM) SE Runtime Environment (build pwa6480sr3fp20ifix-20161110_01(SR3 FP20+I
V90630+IV90578))
IBM J9 VM (build 2.8, JRE 1.8.0 Windows 7 amd64-64 Compressed References 2016101
3_322271 (JIT enabled, AOT enabled)
J9VM - R28_Java8_SR3_20161013_1635_B322271
JIT  - tr.r14.java.green_20161011_125790
GC   - R28_Java8_SR3_20161013_1635_B322271_CMPRSS
J9CL - 20161013_322271)
JCL - 20161018_01 based on Oracle jdk8u111-b14`

Any clue?
I need to have IBM jvm and Team Explorer Everywhere running in the same eclipse

Thanks!

**Comment by madhurig** [comment]
@andreasbisiach: Unfortunately it looks like the version of IBM JRE you are using has a bug or is incompatible with Oracle JRE that Team Explorer Everywhere supports. What version of the IBM jdk/jre are you using? You can try updating to the latest they have and see if that helps.

**Comment by andreasbisiach** [comment]
@madhurig The info was already in the post
java -version shows the following:

C:\IBM\WebSphere\AppServer\java\jre\bin>java.exe -version java version "1.8.0" Java(TM) SE Runtime Environment (build pwa6480sr3fp20ifix-20161110_01(SR3 FP20+I V90630+IV90578)) IBM J9 VM (build 2.8, JRE 1.8.0 Windows 7 amd64-64 Compressed References 2016101 3_322271 (JIT enabled, AOT enabled) J9VM - R28_Java8_SR3_20161013_1635_B322271 JIT - tr.r14.java.green_20161011_125790 GC - R28_Java8_SR3_20161013_1635_B322271_CMPRSS J9CL - 20161013_322271) JCL - 20161018_01 based on Oracle jdk8u111-b14

It should be the latest version

**Comment by madhurig** [comment]
@andreasbisiach : Is it an option to try an older version of the IBM JRE? 
This seems to be an issue with the IBM JRE where it assumes all XML will use a namespace prefix: [similar to this issue](http://stackoverflow.com/questions/11993758/prefix-is-null-error-when-attempting-to-set-up-web-service-client).

**Comment by andreasbisiach** [comment]
@madhurig The short answer is No
IBM WebSphere Application Server 8.5 also ships with IBM Java 1.7
I have tried to modify eclipse.ini to use that but it won't start.
The error I get is "Version 1.7.0 of the JVM is not suitable for this product. Version 1.8 or greater is required"
I guess Eclipse Neon requires 1.8

Would you think that any of the [following parameter](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.win.80.doc/diag/appendixes/cmdline/commands_jvm.html) could mitigate the problem you mentioned in the "similar issue"

**Comment by tomr2017** [comment]
Hello,
i'm experiencing the same kind of issue with the Team Explorer Everywhere (14.114) in eclipse.
2017-04-03 17:01:31,778 INFO  [Worker-5] (com.microsoft.tfs.client.common.commands.QueryTeamProjectsCommand) Querying Team Projects
2017-04-03 17:01:31,794 INFO  [Worker-1] (com.microsoft.tfs.client.common.commands.QueryTeamsCommand) Querying Teams
2017-04-03 17:01:31,841 INFO  [Worker-6] (com.microsoft.tfs.core.internal.db.ConnectionConfiguration) locked [C:\Users\ap4i1\AppData\Local\Microsoft\Team Foundation\4.0\Cache\TEE-WorkItemTracking\4.0A-4fa87d2d-cd63-43f2-89e7-ac1b0777c520\data]
2017-04-03 17:01:32,434 INFO  [Worker-6] (com.microsoft.tfs.client.common.ui.config.UITransportRequestHandler) Authentication requested: 
java.lang.IllegalArgumentException: Das angegebene Präfix war null.
	at com.ibm.xml.xlxp.api.stax.msg.StAXMessageProvider.throwIllegalArgumentException(Unknown Source)
	at com.ibm.xml.xlxp.api.stax.XMLStreamWriterBase.beginStartElement(Unknown Source)
	at com.ibm.xml.xlxp.api.stax.XMLStreamWriterBase.writeStartElement(Unknown Source)
	at com.ibm.xml.xlxp.api.stax.XMLOutputFactoryImpl$XMLStreamWriterProxy.writeStartElement(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.stax.StaxUtils.copyCurrentElement(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.types.StaxAnyContentType.readFromElement(Unknown Source)
	at ms.tfs.workitemtracking.clientservices._03._ClientService5Soap_GetMetadataEx2Response.readFromElement(Unknown Source)
	at ms.tfs.workitemtracking.clientservices._03._ClientService5Soap12Service$34.readSOAPResponse(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequestInternal(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequest(Unknown Source)
	at ms.tfs.workitemtracking.clientservices._03._ClientService5Soap12Service.getMetadataEx2(Unknown Source)
	at com.microsoft.tfs.core.clients.workitem.internal.metadata.Metadata.update(Unknown Source)
	at com.microsoft.tfs.core.clients.workitem.internal.metadata.Metadata.update(Unknown Source)
	at com.microsoft.tfs.core.clients.workitem.internal.WITContext.<init>(Unknown Source)
	at com.microsoft.tfs.core.clients.workitem.WorkItemClient.getContext(Unknown Source)
	at com.microsoft.tfs.core.clients.workitem.WorkItemClient.refreshCache(Unknown Source)
	at com.microsoft.tfs.client.common.server.TFSServer.refresh(Unknown Source)
	at com.microsoft.tfs.client.common.server.TFSServer.access$000(Unknown Source)
	at com.microsoft.tfs.client.common.server.TFSServer$1.run(Unknown Source)
	at org.eclipse.core.internal.jobs.Worker.run(Unknown Source)

Attached you find the diagonse Information:

[tee-diag-04-03-17_17-05-21.zip](https://github.com/Microsoft/team-explorer-everywhere/files/890505/tee-diag-04-03-17_17-05-21.zip)
)

Can you help me with that Problem?

**Comment by jiejiuk1** [comment]
Same here 

```
<?xml version="1.0" encoding="UTF-8"?><error-report locale="en_US" product-id="com.ibm.rational.rad.product.v75.ide" product-name="Rational® Application Developer™ for WebSphere® Software" time="5 Apr 2017 15:40:57 GMT" version="1.0">
<about Version="7.5.0" mapping0="$BUILD_ID" mapping1="7.5.0"/>
<status code="2" message="An internal error occurred during: quot;Refreshing TFS Server information for http://sc0050v:22273/DefaultCollection/quot;." plugin-id="org.eclipse.core.jobs" severity="4">
<trace class="com.ibm.xml.xlxp.api.stax.msg.StAXMessageProvider" line="-1" method="throwIllegalArgumentException"/>
<trace class="com.ibm.xml.xlxp.api.stax.XMLStreamWriterBase" line="-1" method="beginStartElement"/>
<trace class="com.ibm.xml.xlxp.api.stax.XMLStreamWriterBase" line="-1" method="writeStartElement"/>
<trace class="com.ibm.xml.xlxp.api.stax.XMLOutputFactoryImpl$XMLStreamWriterProxy" line="-1" method="writeStartElement"/>
<trace class="com.microsoft.tfs.core.ws.runtime.stax.StaxUtils" line="-1" method="copyCurrentElement"/>
<trace class="com.microsoft.tfs.core.ws.runtime.types.StaxAnyContentType" line="-1" method="readFromElement"/>
<trace class="ms.tfs.workitemtracking.clientservices._03._ClientService5Soap_GetMetadataEx2Response" line="-1" method="readFromElement"/>
<trace class="ms.tfs.workitemtracking.clientservices._03._ClientService5Soap12Service$34" line="-1" method="readSOAPResponse"/>
<trace class="com.microsoft.tfs.core.ws.runtime.client.SOAPService" line="-1" method="executeSOAPRequestInternal"/>
<trace class="com.microsoft.tfs.core.ws.runtime.client.SOAPService" line="-1" method="executeSOAPRequest"/>
<trace class="ms.tfs.workitemtracking.clientservices._03._ClientService5Soap12Service" line="-1" method="getMetadataEx2"/>
<trace class="com.microsoft.tfs.core.clients.workitem.internal.metadata.Metadata" line="-1" method="update"/>
<trace class="com.microsoft.tfs.core.clients.workitem.internal.metadata.Metadata" line="-1" method="update"/>
<trace class="com.microsoft.tfs.core.clients.workitem.internal.WITContext" line="-1" method="lt;initgt;"/>
<trace class="com.microsoft.tfs.core.clients.workitem.WorkItemClient" line="-1" method="getContext"/>
<trace class="com.microsoft.tfs.core.clients.workitem.WorkItemClient" line="-1" method="refreshCache"/>
<trace class="com.microsoft.tfs.client.common.server.TFSServer" line="-1" method="refresh"/>
<trace class="com.microsoft.tfs.client.common.server.TFSServer" line="-1" method="access$000"/>
<trace class="com.microsoft.tfs.client.common.server.TFSServer$1" line="-1" method="run"/>
<trace class="org.eclipse.core.internal.jobs.Worker" line="-1" method="run"/>
</status>
</error-report>
```

**Comment by leantk** [comment]
The IBM JRE has it's own XML parser that does not recognize namespaces automatically (the Oracle JDK does). It looks like Eclipse has an open issue to set the IBM JRE to recognize namespaces since it's an issue for many people but they have not completed the work yet. The Eclipse issue can be found here: https://bugs.eclipse.org/bugs/show_bug.cgi?id=488945

We apologize for the inconvenience but the parser needs this flag to be set to correctly parse the XML.

Thanks,
-Leah

**Comment by andreasbisiach** [comment]
@leantk Thanks for the heads up, but for what I read in the linked bug, the developer writes
"
Thanks for the patch Thomas, I'll consider it for the BPMN2 editor resource base class (Bpmn2ModelerResource) as this problem most likely affects anyone using the IBM JDK parser
" 

I don't know TEE architecture and but reading the above, doesn't that mean that TEE should/could implemente a similar approach?

**Comment by leantk** [comment]
Hi @andreasbisiach,

I tried to make that change in the plugin and made a custom build with it. I've attached it here. Would you mind trying it out to see if there is any difference in behavior?
[TFSEclipsePlugin-UpdateSiteArchive-14.116.0.zip](https://github.com/Microsoft/team-explorer-everywhere/files/917472/TFSEclipsePlugin-UpdateSiteArchive-14.116.0.zip)

Thanks!
-Leah

**Comment by jiejiuk1** [comment]
No difference, same error

**Comment by andreasbisiach** [comment]
Hello @leantk 

I upgraded the plugin with the local update site you provided, I get the same error.

If you provide me instructions to collect information I would be more than happy to help you debugging the problem

**Comment by leantk** [comment]
Thanks for trying @jiejiuk1 and @andreasbisiach. It looks like I might have missed one more place that I needed to make the fix. I'm sorry about that. Would you be able to test this new build one more time please? 
[TFSEclipsePlugin-StaxFix.zip](https://github.com/Microsoft/team-explorer-everywhere/files/922757/TFSEclipsePlugin-StaxFix.zip)

Again my apologies for the trouble,
-Leah

**Comment by jiejiuk1** [comment]
I have just installed it.   When connecting to TFS the following error is displayed multiple times:  _Querying Team Projects:  Unable to handle request without a valid action parameter. Please supply a valid soap action._
And also this one, but only once:    _Reconciling local TFS workspace:  An internal error occurred during: "Reconciling local TFS workspace".
Unable to handle request without a valid action parameter. Please supply a valid soap action._


Here's a partial plugin log.

`2017-04-14 15:27:00,398 INFO  [main] (com.microsoft.tfs.core.telemetry.TfsTelemetryHelper) Plugin for Eclipse v.14.116.0.201704141837
2017-04-14 15:27:00,398 INFO  [main] (com.microsoft.tfs.core.telemetry.TfsTelemetryHelper) AppInsights telemetry initialized
2017-04-14 15:27:00,398 INFO  [main] (com.microsoft.tfs.core.telemetry.TfsTelemetryHelper)     Developer Mode: false
2017-04-14 15:27:00,398 INFO  [main] (com.microsoft.tfs.core.telemetry.TfsTelemetryHelper)     Production Environment: true
2017-04-14 15:27:00,406 INFO  [main] (com.microsoft.tfs.jni.loader.NativeLoader) Loaded native_registry with System.loadLibrary()
2017-04-14 15:27:01,555 INFO  [Worker-7] (com.microsoft.tfs.client.eclipse.project.ProjectRepositoryManager) Starting TFS repository manager
2017-04-14 15:27:01,559 INFO  [Worker-7] (com.microsoft.tfs.client.eclipse.project.ProjectRepositoryManager) Connecting project RADTest to TFS server
2017-04-14 15:27:01,567 INFO  [Worker-8] (com.microsoft.tfs.client.common.ui.autoconnect.UIAutoConnector) Auto connecting to the previously used server: http://sc0050v:22273/
2017-04-14 15:27:01,590 INFO  [Worker-8] (com.microsoft.tfs.client.common.ui.autoconnect.UIAutoConnector) Auto connection is not requested.
2017-04-14 15:27:02,336 INFO  [main] (com.microsoft.tfs.jni.loader.NativeLoader) Loaded native_synchronization with System.loadLibrary()
2017-04-14 15:27:02,375 INFO  [main] (com.microsoft.tfs.jni.loader.NativeLoader) Loaded native_messagewindow with System.loadLibrary()
2017-04-14 15:27:02,431 INFO  [Worker-4] (com.microsoft.tfs.jni.loader.NativeLoader) Loaded native_credential with System.loadLibrary()
2017-04-14 15:27:02,434 INFO  [Worker-4] (com.microsoft.tfs.client.eclipse.project.ProjectManagerConnectionJob) Connecting to server http://sc0050v:22273/DefaultCollection
2017-04-14 15:27:02,475 INFO  [Worker-4] (com.microsoft.tfs.core.config.httpclient.DefaultHTTPClientFactory) HttpClient configured for http://sc0050v:22273/DefaultCollection, authenticating as logged in user, proxy=siproxy2:80
2017-04-14 15:27:02,588 INFO  [Build Status Manager] (com.microsoft.tfs.client.common.server.cache.buildstatus.BuildStatusManager$BuildStatusManagerRefreshWorker) Starting build status refresh worker
2017-04-14 15:27:02,724 INFO  [Worker-1] (com.microsoft.tfs.jni.loader.NativeLoader) Loaded native_filesystem with System.loadLibrary()
2017-04-14 15:27:02,739 INFO  [Worker-2] (com.microsoft.tfs.client.eclipse.project.ProjectRepositoryManager) Connection to TFS server successful for project RADTest
2017-04-14 15:27:02,754 INFO  [Worker-9] (com.microsoft.tfs.client.common.commands.QueryTeamProjectsCommand) Querying Team Projects
2017-04-14 15:27:02,758 INFO  [Worker-8] (com.microsoft.tfs.client.common.commands.QueryTeamProjectsCommand) Querying Team Projects
2017-04-14 15:27:03,015 INFO  [TFS Connection] (com.microsoft.tfs.jni.loader.NativeLoader) Loaded native_auth with System.loadLibrary()
2017-04-14 15:27:03,149 INFO  [TFS Connection] (com.microsoft.tfs.client.common.ui.config.UITransportRequestHandler) Authentication requested: 
com.microsoft.tfs.core.ws.runtime.exceptions.SOAPFault: Unable to handle request without a valid action parameter. Please supply a valid soap action.
	at com.microsoft.tfs.core.ws.runtime.client.SOAP12Service.examineResponseDOMForFault(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.examineBodyForFault(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequestInternal(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequest(Unknown Source)
	at ms.ws._LocationWebServiceSoap12Service.connect(Unknown Source)
	at com.microsoft.tfs.core.clients.framework.location.internal.LocationWebServiceProxy.connect(Unknown Source)
	at com.microsoft.tfs.core.clients.framework.location.LocationService.connect(Unknown Source)
	at com.microsoft.tfs.core.clients.framework.location.LocationService.authenticate(Unknown Source)
	at com.microsoft.tfs.core.TFSConnection.authenticate(Unknown Source)
	at com.microsoft.tfs.client.common.commands.TFSConnectedCommand$TFSCommandConnectionRunnable.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2017-04-14 15:27:03,159 ERROR [Worker-8] (com.microsoft.tfs.client.common.commands.QueryTeamProjectsCommand) Unable to handle request without a valid action parameter. Please supply a valid soap action. (Unable to handle request without a valid action parameter. Please supply a valid soap action.)
com.microsoft.tfs.core.exceptions.TECoreException: Unable to handle request without a valid action parameter. Please supply a valid soap action.
	at com.microsoft.tfs.core.exceptions.mappers.TECoreExceptionMapper.map(Unknown Source)
	at com.microsoft.tfs.core.exceptions.mappers.LocationExceptionMapper.map(Unknown Source)
	at com.microsoft.tfs.core.clients.framework.location.internal.LocationWebServiceProxy.connect(Unknown Source)
	at com.microsoft.tfs.core.clients.framework.location.LocationService.connect(Unknown Source)
	at com.microsoft.tfs.core.clients.framework.location.LocationService.authenticate(Unknown Source)
	at com.microsoft.tfs.core.TFSConnection.authenticate(Unknown Source)
	at com.microsoft.tfs.client.common.commands.TFSConnectedCommand$TFSCommandConnectionRunnable.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: 
com.microsoft.tfs.core.ws.runtime.exceptions.SOAPFault: Unable to handle request without a valid action parameter. Please supply a valid soap action.
	at com.microsoft.tfs.core.ws.runtime.client.SOAP12Service.examineResponseDOMForFault(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.examineBodyForFault(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequestInternal(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequest(Unknown Source)
	at ms.ws._LocationWebServiceSoap12Service.connect(Unknown Source)
	... 6 more
2017-04-14 15:27:03,160 ERROR [main] (com.microsoft.tfs.client.common.server.cache.project.ServerProjectCache) Could not update team project list: Unable to handle request without a valid action parameter. Please supply a valid soap action.
2017-04-14 15:27:03,161 INFO  [Worker-6] (com.microsoft.tfs.client.common.commands.QueryTeamProjectsCommand) Querying Team Projects
2017-04-14 15:27:03,171 INFO  [Worker-0] (com.microsoft.tfs.client.common.ui.config.UITransportRequestHandler) Authentication requested: 
com.microsoft.tfs.core.ws.runtime.exceptions.SOAPFault: Unable to handle request without a valid action parameter. Please supply a valid soap action.
	at com.microsoft.tfs.core.ws.runtime.client.SOAP12Service.examineResponseDOMForFault(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.examineBodyForFault(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequestInternal(Unknown Source)
	at com.microsoft.tfs.core.ws.runtime.client.SOAPService.executeSOAPRequest(Unknown Source)
	at ms.tfs.versioncontrol.clientservices._03._Repository4Soap12Service.queryPendingChangeSignature(Unknown Source)
	at com.microsoft.tfs.core.clients.versioncontrol.internal.WebServiceLayer.queryPendingChangeSignature(Unknown Source)
	at com.microsoft.tfs.core.clients.versioncontrol.internal.localworkspace.LocalDataAccessLayer$8.invoke(Unknown Source)
	at com.microsoft.tfs.core.clients.versioncontrol.internal.localworkspace.LocalWorkspaceTransaction.execute(Unknown Source)
	at com.microsoft.tfs.core.clients.versioncontrol.internal.localworkspace.LocalDataAccessLayer.reconcileLocalWorkspaceHelper(Unknown Source)
	at com.microsoft.tfs.core.clients.versioncontrol.internal.localworkspace.LocalDataAccessLayer.reconcileLocalWorkspace(Unknown Source)
	at com.microsoft.tfs.core.clients.versioncontrol.soapextensions.Workspace.reconcile(Unknown Source)
	at com.microsoft.tfs.client.eclipse.ui.TFSEclipseClientUIPlugin$RepositoryListener$1.run(Unknown Source)
	at org.eclipse.core.internal.jobs.Worker.run(Unknown Source)`


Let me know if you need the whole thing, it's pretty long

**Comment by andreasbisiach** [comment]
@leantk I am getting exactly the same error as jiejiuk1

**Comment by leantk** [comment]
Ok thanks for trying! We need to look into this further then since it wasn't as easy of a fix as that bug forum made it sound. I have it on our backlog to tackle and we will update accordingly here. 
Sorry again for the trouble,
-Leah

**Comment by andreasbisiach** [comment]
@leantk 

No problem, just let me know when you need more testing for a new build and if you need me to turn on special debugging parameters

**Comment by AlexRukhlin** [comment]
This is a private build that supposed to resolve the issue: [TFSEclipsePlugin-UpdateSiteArchive-14.116.0.zip](https://github.com/Microsoft/team-explorer-everywhere/files/970879/TFSEclipsePlugin-UpdateSiteArchive-14.116.0.zip). Could you give it a try?

**Comment by andreasbisiach** [comment]
@AlexRukhlin 
Thanks a lot! I have just installed it and given a quick test. Seems all issues are corrected!

Nice job!

**Comment by andreasbisiach** [comment]
@AlexRukhlin I was wondering, when will this be merged into the official repository and how can I verify it?

I would like to inform my customers currently having the issue

Thanks
